Formula/c/chezmoi.rb

```diff
@@ -28,7 +28,7 @@ def install
     ldflags = %W[
       -s -w
       -X main.version=#{version}
-      -X main.commit=#{tap.user}
+      -X main.commit=#{Utils.git_head}
```

Comment:
The commit is usually not required for Homebrew builds, because bottles are always built from a tag.

`Utils.git_head` is not available at build time unless we checkout the repo, which is the less preferred method to using the release archive.

Reply:
it is not needed to display both commit sha and the version (as they are the same thing) for a homebrew release build.

Reply:
It may be worth doing this for HEAD builds which are built from latest `master` commit. 

We should probably document our choice of setting main.commit somewhere at https://docs.brew.sh, maybe the Cookbook

---

On side note, it is actually possible to get this from archive since `git archive` embeds it in PAX headers, e.g.
```console
❯ curl -sL https://github.com/twpayne/chezmoi/archive/refs/tags/v2.60.1.tar.gz | gzcat | git get-tar-commit-id
1f95b695620669fd4a11f779df03f7fd73203dee
```

Would probably require `brew` support if we ever want this since it is probably best to obtain this while unpacking.

Reply:
chezmoi also includes a file called `COMMIT` in its archive tarballs containing the git commit of the release.